openstef_dbc/data_interface.py

```diff
@@ -210,7 +217,7 @@ def exec_sql_query(self, query, **kwargs):
             self.logger.error("Can't connecto to MySQL database", exc_info=e)
             raise
 
-    def exec_sql_write(self, statement):
+    def exec_sql_write(self, statement: str) -> None:
         try:
             with self.mysql_engine.connect() as connection:
                 connection.execute(statement)
```

Comment:
This function still executes a raw query. This is potential risk. Does each reference to this function have proper custom parameterization?

Reply:
Good point, this also relates to your commenbt about teh write_location method